docker-compose.yml

```diff
@@ -0,0 +1,41 @@
+version: '2'
+services:
+  web:
+    restart: always
+    build: ./server/web
+    ports:
+      - "8000:8000"
+    links:
+      - mysql:mysql
+    volumes:
+      - ./server/web/code:/code
+  
+  nginx:
+    restart: always
+    image: nginx:latest
+    ports:
+      - "80:80"
+    volumes:
+      - ./server/nginx:/usr/share/nginx/html
```

Comment:
I believe the nginx volume should mount client/dist since that's where the front end files are located